### PR TITLE
Feature: Created Simplified grid component

### PR DESCRIPTION
**Pull request by poojakelkar**
- Added Grid component.
- Added Product and price label in Grid.

Live Link: https://quizzical-bardeen-c6adc0.netlify.app/

### CODE REVIEW

File: components/simplified-grid/styles.css [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    color: var(--font-color);
+}
+.add {
+    border-radius: 50%;
+    border: 1px solid black;
+    padding: 0.2rem 0.55rem;
+    margin: 0.50rem;
+}
+.quantity-total {
+    border-radius: 0.45rem;
```
Comment (tushargoyal812) [inline components/simplified-grid/styles.css:54]: use pixels for border and border radius

**Review by tushargoyal812 — COMMENTED** [review]
take care of suggestion,looks good to merge

File: components/simplified-grid/styles.css (hunk 1 above)
Comment (poojakelkar) [reply components/simplified-grid/styles.css:54]: sure Thanks for suggestion

**Review by shobit1337 — APPROVED** [review]
Looks good to me!

**Comment by poojakelkar** [comment]
Thanks shobit

**Pull request merged**